hathor/simulator/simulator.py

```diff
@@ -189,7 +189,7 @@ def add_connection(self, conn: 'FakeConnection') -> None:
         self._connections.append(conn)
 
     def _run(self, interval: float, step: float, status_interval: float) -> Generator[None, None, None]:
-        """ Implementation of run, yields at every step to allow verifications like in run_until_synced
+        """ Implementation of run, yields at every step to allow verifications like in run_until_synced_or_error
```

Comment:
Outdated comment. Need to update `run_until_synced_or_error` to `run_until_complete`.

Reply:
Fixed.

Reply:
Thanks!